Components/badge.css

```diff
@@ -0,0 +1,32 @@
+.img{
+    border-radius: 50%;
+    width:90px;
+    height: 90px;
+    object-fit: cover;
+}
+.parent{
+    position:relative;
+}
+.icon-cart{
+    font-size:30px;
+}
+.icon{
+    font-size:18px;
+    position:absolute;
+}
+.icon1-2{
+    padding:3px;
+    bottom:0;
+    left:60px;
+    color: var(--green-shade-color2)
+}
+.icon-pink{
+    color: var(--pink-shade-color)
+}
+.icon3{
+    top:-10px;
+    right:-10px;
+    color:var(--green-shade-color1);
+}
```

Comment:
I would say that use good naming format because icon1-2 I am unable to understand that whether it is icon size or different icon format. Maybe use anything other that numbers inside icon. Rest I think its good to go.